servers/rendering/renderer_rd/shaders/scene_forward_mobile.glsl

```diff
@@ -650,7 +650,7 @@ void main() {
 
 	float normal_map_depth = 1.0;
 
-	vec2 screen_uv = gl_FragCoord.xy * scene_data.screen_pixel_size + scene_data.screen_pixel_size * 0.5; //account for center
+	vec2 screen_uv = gl_FragCoord.xy * scene_data.screen_pixel_size; //account for center
```

Comment:
The comment wasn't removed, is that still correct?

Reply:
No, I've removed it now.